Pod/Classes/Internal/UIViewController+SEGScreen.m

```diff
@@ -0,0 +1,76 @@
+#import "UIViewController+SEGScreen.h"
+#import <objc/runtime.h>
+#import "SEGAnalytics.h"
+
+
+@implementation UIViewController (SEGScreen)
+
++ (void)swizzleViewDidAppear
+{
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        Class class = [self class];
+
+        SEL originalSelector = @selector(viewDidAppear:);
+        SEL swizzledSelector = @selector(screen_viewDidAppear:);
+
+        Method originalMethod = class_getInstanceMethod(class, originalSelector);
+        Method swizzledMethod = class_getInstanceMethod(class, swizzledSelector);
+
+        BOOL didAddMethod =
+            class_addMethod(class,
+                            originalSelector,
+                            method_getImplementation(swizzledMethod),
+                            method_getTypeEncoding(swizzledMethod));
+
+        if (didAddMethod) {
+            class_replaceMethod(class,
+                                swizzledSelector,
+                                method_getImplementation(originalMethod),
+                                method_getTypeEncoding(originalMethod));
+        } else {
+            method_exchangeImplementations(originalMethod, swizzledMethod);
+        }
+    });
+}
+
+
++ (UIViewController *)topViewController
```

Comment:
We should prefix our method names if we were to add category to global classes such as `UIViewController`. something like `seg_topViewController`